plugins/modules/notification/cisco_webex.py

```diff
@@ -10,99 +10,103 @@
 
 DOCUMENTATION = '''
 ---
-module: cisco_spark
-short_description: Send a message to a Cisco Spark Room or Individual.
+module: cisco_webex
+short_description: Send a msg to a Cisco Webex Teams Room or Individual
 description:
-    - Send a message to a Cisco Spark Room or Individual with options to control the formatting.
+    - Send a msg to a Cisco Webex Teams Room or Individual with options to control the formatting.
+version_added: "2.10"
 author: Drew Rusell (@drew-russell)
 notes:
   - The C(recipient_id) type must be valid for the supplied C(recipient_id).
-  - Full API documentation can be found at U(https://developer.ciscospark.com/endpoint-messages-post.html).
+  - Full API documentation can be found at U(https://developer.webex.com/docs/api/basics).
 
 options:
 
   recipient_type:
     description:
-       - The request parameter you would like to send the message to.
+       - The request parameter you would like to send the msg to.
        - Messages can be sent to either a room or individual (by ID or E-Mail).
-    required: True
+    required: yes
     choices: ['roomId', 'toPersonEmail', 'toPersonId']
+    type: str
 
   recipient_id:
     description:
       - The unique identifier associated with the supplied C(recipient_type).
-    required: true
+    required: yes
+    type: str
 
-  message_type:
+  msg_type:
     description:
-       - Specifies how you would like the message formatted.
-    required: False
+       - Specifies how you would like the msg formatted.
     default: text
     choices: ['text', 'markdown']
+    type: str
 
   personal_token:
     description:
-      - Your personal access token required to validate the Spark API.
-    required: true
-    aliases: ['token']
+      - Your personal access token required to validate the Webex Teams API.
+    required: yes
+    type: str
 
-  message:
+  msg:
     description:
-      - The message you would like to send.
-    required: True
+      - The msg you would like to send.
```

Comment:
```suggestion
      - The message you would like to send.
```
if the parameter's name "msg" stands for "message", i think, it would be better to left it as it is

Reply:
(in the description)